src/WebRtcTransport.ts

```diff
@@ -16,7 +16,7 @@ export type WebRtcTransportOptions =
 	 * Listening IP address or addresses in order of preference (first one is the
 	 * preferred one).
 	 */
-	listenIps: TransportListenIp[] | string[];
+	listenIps: Array<TransportListenIp | string>;
```

Comment:
Why this change? It was on purpose. I really prefer to use `[]` syntax rather than `Array`.

Reply:
This is a more correct way of typing such things as it allows you to use array methods on `listenIps` without casting it first with `(listenIps as any[])` and loosing type information in the process.